src/Execution/Arguments/UpsertModel.php

```diff
@@ -22,20 +26,39 @@ public function __construct(callable $previous)
     public function __invoke($model, $args): mixed
     {
         // TODO consider Laravel native ->upsert(), available from 8.10
```

Comment:
I wonder if Laravel upsert() does what we need already and we unnecessarily diverge?

Reply:
I looked into this, unfortunately not, Laravel's upsert simply bulk inserts raw data into the DB bypassing the model events, casts, setters etc. Probably not what most users want here. I think the Laravel 'upsert' is more inline with the database layer concept of upsert and not the graphql mutation upsert concept.